torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -945,3 +957,8 @@ class AMXState {
     tile_release();
   }
 };
+
+// For group-wise quantization, e.g., WOQ INT4
+inline bool is_block_start(int index, int k_start, int group_size) {
+  return (k_start + index) % group_size == 0;
+}
```

Comment:
This is specific to gemm template, better not put it in cpp_prefix but in template codegen.

Reply:
Thanks. I have moved this to microkernel code.